test/e2e/sched/install/install_test.go

```diff
@@ -96,6 +96,7 @@ var _ = Describe("[Scheduler] install", func() {
 			By("Check secondary scheduler pod is scheduled on a control-plane node")
 			podList, err := podlist.ByDeployment(e2eclient.Client, *deployment)
 			Expect(err).NotTo(HaveOccurred())
+			Expect(podList).ToNot(BeEmpty(), "cannot find any pods for DS %s/%s", deployment.Namespace, deployment.Name)
```

Comment:
here and below , if i'm not mistaken it should  be DP (deployment) instead of DS (daemonset)

Reply:
right, thanks! will fix